### DISCUSSION TITLE
Code coverage with nyc?

### DISCUSSION THREAD

**Discussion by compojoom** [wix/Detox#2956] [Q&A]
I just ran the detox tests in the detox/test folder and I was astonished to see that they seem to generate a test coverage report. 
What do I have to generate test coverage for any other project using detox? As far as I could see from the code we have to install `nyc` and use it as test runner `nyc jest`. 

I did that and i also modified our config, but the generate nyc report is empty. I tried playing with the .nycrc file, but the best I could achieve was to list the babel.config.js file?

Is there anything special that needs to be configured in detox to enable code coverage?

**Comment by d4vidi** [comment]
@noomorph could you advise?
@compojoom could you elaborate - coverage over which code are you looking for (the app's)? If so, it is far from trivial, because Detox's test code runs in an entirely different context (i.e. a `node.js` instance on your local computer).
